nss-day-cohort-43/nuttyshell-the-coven-code
# At events

# Description
1- Add the events section to the dashboard. The add events + and the delete X are functional. Weather and edit are separate issue tickets.
2- Added images of wireframe to our readme file

## Type of change
-[x] New feature (non-breaking change which adds functionality)

# Testing Instructions
See if event list for Active User (if info is already in the database) is rendered when the dashboard appears. Try to add an event with the plus sign. Try to delete an event.

# Checklist:
